packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -223,7 +223,8 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
         assert.strictEqual(map2.get(testKey), bigString);
     });
 
-    it("doesn't resend successful chunked op", async function() {
+    // TODO: investigate reenabling this test if still relevant without a BatchManager
```

Comment:
reference to issue is better than TODO comment.

Reply:
I agree. The reason I left it as todo is to finish the investigation BEFORE making this PR ready for review. This test also has some issues with t9s.

Reply:
As long as we don't do chunking, this is irrelevant.